lib/init/rc.boot

```diff
@@ -117,7 +117,7 @@ main() {
     }
 
     log "Mounting all local filesystems..."; {
-        mount -at nosysfs,nonfs,nonfs4,nosmbfs,nocifs -O no_netdev ||
+        mount -a ||
```

Comment:
This should be fine. :+1:

Reply:
The message above should be changed to `Mounting all filesystems...` as the removal of these flags will cause `mount` to attempt to mount network partitions. I've never actually seen this use-case so we should be fine here.